api/mastodon.ts

```diff
@@ -0,0 +1,26 @@
+import got from '../libs/got'
+import { millify } from '../libs/utils'
+import { createBadgenHandler, PathArgs } from '../libs/create-badgen-handler'
+
+export default createBadgenHandler({
+  title: 'Mastodon/Pleroma',
+  examples: {
+    '/mastodon/follow/1': 'followers count (mastodon.social)',
+    '/mastodon/follow/1/mas.to': 'followers count (custom Mastodon instance)',
+    '/mastodon/follow/1/pleroma.site': 'followers count (custom Pleroma instance)',
```

Comment:
It would be nice to make the description shorter, avoid that overflow in screenshot :)

Reply:
![image](https://user-images.githubusercontent.com/1170440/82597285-042c1e80-9ba9-11ea-9c30-6b3a46e8ed40.png)

Better? 👨‍🎨

Reply:
Looks nice!